go/mysql/mysql_throttle_metric.go

```diff
@@ -45,6 +73,10 @@ func ReadThrottleMetric(probe *Probe) (mySQLThrottleMetric *MySQLThrottleMetric)
 		}()
 	}(mySQLThrottleMetric, started)
 
+	if mySQLThrottleMetric := getCachedMySQLThrottleMetric(probe); mySQLThrottleMetric != nil {
+		return mySQLThrottleMetric
+	}
+
```

Comment:
Because now cache lookups will be faster than running the `MetricQuery`, this would change the `probes.latency` metric.
Should you create a special metric for cache hits/misses?

Reply:
Good observation. I will!

Reply:
A committed change avoids latency metrics altogether on cached results. I think cache latency measurements are not very interesting, coming from a local `golang` cache. I'm OK to nto measure them.